### PR TITLE
MC option for Xic0 semileptonic mode

### PR DESCRIPTION
**Pull request by sh-lim**
Add the 4body decay mode for the Xic0 semileptonic simulation

### CODE REVIEW

**Review by chiarazampolli — APPROVED** [review]

File: PYTHIA8/AliPythia8/AliDecayerPythia8.cxx [hunk 1]
```diff
@@ -865,6 +867,13 @@ void AliDecayerPythia8::ForceHadronicD(Int_t optUse4Bodies, Int_t optUseDtoV0, I
     }       
     else if (optForceXicChannel == 1) { // semileptonic decay
         fPythia8->ReadString("4132:onIfMatch = 3312 11 12");
+        fPythia8->ReadString("4132:onIfMatch = 3314 11 12");
+				fPythia8->ReadString("4232:onMode = off");
```
Comment (chiarazampolli) [inline PYTHIA8/AliPythia8/AliDecayerPythia8.cxx:871]: Here you might need to adjust indentation.

File: PYTHIA8/AliPythia8/AliDecayerPythia8.cxx [hunk 2]
```diff
@@ -865,6 +867,13 @@ void AliDecayerPythia8::ForceHadronicD(Int_t optUse4Bodies, Int_t optUseDtoV0, I
...
+        fPythia8->ReadString("4232:onIfMatch = 3324 11 12");
+				fPythia8->ReadString("3314:onMode = off");
```
Comment (chiarazampolli) [inline PYTHIA8/AliPythia8/AliDecayerPythia8.cxx:873]: Same here.

File: PYTHIA8/AliPythia8/AliDecayerPythia8.cxx [hunk 3]
```diff
@@ -865,6 +867,13 @@ void AliDecayerPythia8::ForceHadronicD(Int_t optUse4Bodies, Int_t optUseDtoV0, I
...
+				fPythia8->ReadString("3324:onMode = off");
```
Comment (chiarazampolli) [inline PYTHIA8/AliPythia8/AliDecayerPythia8.cxx:874]: and here.

Comment (sh-lim) [reply PYTHIA8/AliPythia8/AliDecayerPythia8.cxx]: Fixed indentations.

**Comment by chiarazampolli** [comment]
Hello @TimoWilken , 
Same here, as in https://github.com/alisw/AliRoot/pull/1455 and https://github.com/alisw/AliRoot/pull/1454 ?
Chiara

**Comment by chiarazampolli** [comment]
+1

**Pull request merged**